README.md

```diff
@@ -128,6 +128,17 @@ pipenv run python manage.py load_table --findingtext
 
 In the deployed environment, `django-apscheduler` is used to refresh all tables daily at 12:00 AM EST.
 
+### requirements.txt
+
+To install the spacy English language pack (`en_core_web_sm`) on cloud.gov, we must explicitly specify this dependency as a file download in `requirements.txt` (see this [related spacy Github issue](https://github.com/explosion/spaCy/issues/3536 "spacy Github issue about installation of language models from pyPI"). Unfortunately, this also means maintaining Pipenv _and_ a requirements file.
```

Comment:
Could you accomplish the same by putting this in the `Pipfile`?

```ini
en-core-web-sm = {file = "https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.1.0/en_core_web_sm-2.1.0.tar.gz"}
```

Reply:
Yep, we already have this line in the `Pipfile` (although for version 2.2.0). And this doesn't work: I think cloud.gov generates the requirements.txt itself if the file doesn't exist, and the resulting requirements.txt is not set up correctly. Maybe there's a solution if we tweak the buildpack but that is a bit too far down the rabbit hole for me...

Here's what happens if I push without `requirements.txt`.

```bash
$ rm requirements.txt
$ cf push
.... a lot of text ... 
          Generating 'requirements.txt' from Pipfile.lock
... more text ...
            ERROR: Could not find a version that satisfies the requirement en-core-web-sm (from -r /tmp/app/requirements.txt (line 35)) (from versions: none)
          ERROR: No matching distribution found for en-core-web-sm (from -r /tmp/app/requirements.txt (line 35))
   Failed to compile droplet: Failed to run all supply scripts: exit status 14
   Exit status 223
```

Reply:
Huh, weird one 🤕